### DISCUSSION TITLE
Add option to disable new scrolling feature of Cline in settings.

### DISCUSSION THREAD

**Discussion by AIHypnosis** [cline/cline#585] [Feature Requests]
The new feature when response streamed with a yellow text decoration animation is very confusing and time consuming.
Even in a small file of about 300 lines you have to wait forever for this feature to complete scrolling. 
The problem is that the bot scrolls through the whole file, line by line, not just the block of code that needs to be changed. Also, this makes it impossible to undo the change if something went wrong with the code. This feature is a move in the wrong direction! 

Please add an option in settings to make it possible to disable this feature, fall back to the old Claude Dev behavior.

**Comment by mbleigh** [comment]
+1 it's impossible to review the code as it's being written because the line is constantly moving. The user should retain control of scrolling while Cline is writing the code, allowing them to review / modify the top part of the file before it's completely rewritten.

**Comment by blistovmhz** [comment]
Um... Is this a "feature" ? It seem... like... not a feature. 
I just started testing Cline right now and ... It's not useable whatsoever with regards to edits. It's doing a great job in discussion, but it seems to have zero ability to edit a file. Just scrolls to about half way through the file (brutally slowly) and then craps out.

**Comment by dantess** [comment]
Same here. Also on remote machine it sometimes VERY slow. Any workaround to disable it or at least disable scrolling

**Comment by yubingjiaocn** [comment]
Same here, seems cline is writing to file line by line, rather then write the whole file at once or replace the necessary part.
